jsond/__init__.py

```diff
@@ -39,7 +39,7 @@ def str_to_datetime(self, s):
             return s
 
     def _decode(self, data):
-        if isinstance(data, basestring) and data.startswith('datetime:'):
+        if isinstance(data, str) and data.startswith('datetime:'):
```

Comment:
Is this what we want?
For one, it will commit us to python3:

```
>>> isinstance("hello", str)
True
>>> isinstance(u"hello", str)
False
>>> 
```

In python3 it will be fine, as long as we don't get a bytestring somehow!

```
>>> isinstance("hello", str)
True
>>> isinstance(b"hello", str)
False
```